lite/examples/sound_classification/ios/SoundClassification/GMLAudio/GMLAudio.h

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "GMLRingBuffer.h"
+#import "GMLAudioRecord.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** A wrapper class to store input audio used in on-device machine learning. */
+NS_SWIFT_NAME(Audio)
+@interface GMLAudio : NSObject
+
+/** Audio format specifying the number of channels and sample rate supported. */
+@property(nonatomic, readonly) GMLAudioFormat *audioFormat;
+
+/**
+ * Initializes GMLAudio with a GMLAudioFormat and sample countl.
+ *
+ * @discussion The created instance stores data in a ring buffer of size sampleCount *
+ * GMLAudioFormat.channelCount.
+ *
+ * @param format An audio format of type GMLAudioFormat.
+ * @seealso GMLAudioFormat
+ *
+ * @param sampleCount The number of samples this GMLAudio instance can store at any given time. The
+ * sampleCount provided will be used to calculate the buffer size needed by multiplying with
+ * channelCount of format.
+ *
+ * @return An instance of GMLAudioFormat
+ */
+- (instancetype)initWithAudioFormat:(GMLAudioFormat *)format sampleCount:(NSUInteger)sampleCount;
+
+/**
+ * Convenience method to load the audio record buffer output by a GMLAudioRecord.
+ *
+ * @seealso GMLAudioRecord
+ *
+ * @discussion You must make sure that the buffer size and audio format of GMLAudioRecord matches
+ * the the format.
+ *
+ * @param audioRecordBuffer  A buffer of type GMLFloatBuffer output by GMLAudioRecord. You must make
+ * sure that the buffer size and audio format of GMLAudioRecord matches the the format.
+ *
+ * @return An instance of GMLAudioFormat
+ */
+- (BOOL)loadAudioRecordBuffer:(GMLFloatBuffer *)audioRecordBuffer
```

Comment:
Please change to `loadFromAudioRecord:(TFLAudioRecord *)audioRecord withError:(NSError **)error` to align with the Python and Android APIs. Please also see the comments in TFLAudioRecord.